gnucash/import-export/test/gmock-Account.cpp

```diff
@@ -0,0 +1,114 @@
+#include <config.h>
+
+#include "gmock-Account.h"
+
+
+struct _MockAccountClass
+{
+    QofInstanceClass parent_class;
+};
+typedef struct _MockAccountClass MockAccountClass;
+
+G_DEFINE_TYPE(MockAccount, gnc_mock_account, QOF_TYPE_INSTANCE);
+
+static void
+gnc_mock_account_init (MockAccount *inst)
+{
+    // function is unused, initialization is done in the MockAccount's constructor
```

Comment:
Of course these two functions are called during GObject instantiation. The comment should say, that the C++ ctor is used for instance initialization instead of this function.

Is it clear enough, when I change "MockAccount's constructor" to "MockAccount's C++ constructor" then.

Reply:
I think so.

Reply:
Done in PR #738